main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,56 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in the ("NEON_SVE bridge")
+<../acle.rst#neon-sve-bridge>. These intrinsics add a
+convienient way to use NEON and SVE vectors interchangeably in C code with
```

Comment:
On a strict reading of this, you cannot interchange an SVE and a NEON register in an arbitrary context, so I think you intend to write something different here.
```suggestion
convienient way to convert between NEON and SVE vectors in C code with
```

Reply:
done